spring-kafka/src/main/java/org/springframework/kafka/config/AbstractKafkaListenerContainerFactory.java

```diff
@@ -226,8 +226,9 @@ public void setReplyTemplate(KafkaTemplate<?, ?> replyTemplate) {
 
 	/**
 	 * Set the {@link CommonErrorHandler} which can handle errors for both record and
-	 * batch listeners. Replaces the use of
-	 * {@link org.springframework.kafka.listener.GenericErrorHandler}s.
+	 * batch listeners. Replaces the use of <a href="https://javadoc.io/doc/org.springframework.kafka/
+	 * spring-kafka/latest/org/springframework/kafka/3.0.12/CommonErrorHandler.html">
+	 * {@code org.springframework.kafka.listener.GenericErrorHandler}</a>s.
```

Comment:
I would just stop after `batch listeners.`.
No need to mention a removed code any more and also links for versioned Javadocs are also not OK since versions have a trend to be updated every month.